Creacion de control para poder ingresar datos

Wilmer007 opened the pull request:
Se creo un apartado en el que se pudieron ingresar los datos de un usuario con la informacion requerida

Wilmer007 committed:
Creacion de control para poder ingresar datos

Se creo un apartado en el que se pudieron ingresar los datos de un usuario con la informacion requerida

UntalivanCruz commented on TAREA1GRUPO3/EventoController.cs:
```diff
@@ -24,10 +24,19 @@ public class EventoController : Controller
     };
 
-    [HttpGet("eventos")] 
+    [HttpGet("eventos")]
```
puedes usar `[HttpGet("[Action]")]`

UntalivanCruz commented on TAREA1GRUPO3/obj/Debug/net9.0/TAREA1GRUPO3.AssemblyInfo.cs:
```diff
@@ -13,7 +13,7 @@
 [assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
 [assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
-[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0+58a14d022d6687f2599ba127cb666841cac59bb4")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0+8106030ac85c82164eef3ba0c81e50d21e25a870")]
```
es bueno crear un archivo .gitignore en la raíz del repositorio, hay cosas que no son necesarias subirlas a grthub

Wilmer007 committed:
Se creo el git ignore file

Se creo un File para poder reducir la cantidad de archivos que se actualizan al subirlos a los commit que se envien

UntalivanCruz commented:
No se aplicaron las correcciones, no se subió el archivo .gitignore

UntalivanCruz commented on TAREA1GRUPO3/EventoController.cs:
```diff
@@ -42,10 +51,23 @@ public IActionResult Evento([FromRoute] int id)
+        return Ok(evento);
+    }
+
+    [HttpPost("eventos")]
```
Ya el controller tiene el nombre de /eventos... aqui no seria necesario duplicarlo, se entiende que las acciones CRUD el método Create es el principal del controller, dejar solo `[HttpPost]`
